charts/dapr/README.md

```diff
@@ -92,9 +92,10 @@ The Helm chart has the follow configuration options that can be supplied:
 | `global.dnsSuffix`                        | Kuberentes DNS suffix                                                   | `.cluster.local`        |
 | `global.daprControlPlaneOs`               | Operating System for Dapr control plane                                 | `linux`                 |
 | `global.daprControlPlaneArch`             | CPU Architecture for Dapr control plane                                 | `amd64`                 |
-| `global.nodeSelector`                     | Pods will be scheduled onto a node node whose labels match the nodeSelector | `{}`                 |
-| `global.tolerations`                      | Pods will be allowed to schedule onto a node whose taints match the tolerations | `{}`                 |
-| `global.labels`                           | Custom pod levels                                                       | `{}`                 |
+| `global.nodeSelector`                     | Pods will be scheduled onto a node node whose labels match the nodeSelector | `{}`                |
+| `global.tolerations`                      | Pods will be allowed to schedule onto a node whose taints match the tolerations | `{}`            |
+| `global.labels`                           | Custom pod levels                                                       | `{}`                    |
+| `global.issuerFilenames`                  | Sentry-related certificates filenames                                   | `{}`                    |
```

Comment:
Can you please document the exact fields explicitly?

- `global.issuerFilenames.ca`: name of the file containing the root CA certificate inside the container
- `global.issuerFilenames.cert`: name of the file containing the leaf certificate inside the container
- `global.issuerFilenames.key`: name of the file containing the leaf certificate's key inside the container